src/Mesh/Meshing.jl

```diff
@@ -30,7 +40,8 @@ function generate_mesh(proj::Project)
 
     saveProject(proj)
     fileName       = joinpath(proj.projectDirectory,proj.name)*".control"
-    mesherOutput   = generate_mesh(fileName, output_directory = proj.projectDirectory)
+    mesherOutput   = generate_mesh(fileName, output_directory = proj.projectDirectory,
+                                   verbose=verbose, subdivision_maximum=subdivision_maximum)
```

Comment:
```suggestion
    mesherOutput   = generate_mesh(fileName; output_directory = proj.projectDirectory,
                                   verbose, subdivision_maximum)
```
Optional for brevity, but not strictly required 😊

Reply:
I like the brevity, especially because the variable names are the same across both functions. There should be no confusion as to what is being passed.